Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -335,82 +348,82 @@ private static void Clean(NugetPackageIdentifier package)
 
                     // Select the highest .NET library available that is supported
                     // See here: https://docs.nuget.org/ndocs/schema/target-frameworks
-                    if (usingStandard2 && directoryName == "netstandard2.0")
+                    if (DotNetVersion == ApiCompatibilityLevel.NET_Standard_2_0 && directoryName == "netstandard2.0")
```

Comment:
In the comment above we were using the hardcoded values to support older versions of Unity. Are you sure this hasn't broken 5.3?

Reply:
It's fixed changes to the original.